tests/test.js

```diff
@@ -216,28 +215,25 @@ test.serial('test activity success', async (t) => {
   sf.restore();
 });
 
-test.serial('test activity failure', async (t) => {
+test.serial('test activity failure', async(t) => {
   const input = {
     msg: 'this was a failure',
-    error: {
-      name: 'failure',
-      message: 'it failed'
-    }
+    error: 'it failed'
   };
   const token = 'some token';
 
   const sf = sinon.stub(AWS, 'StepFunctions')
     .returns({
       getActivityTask: () => ({
-        promise: async () => ({
+        promise: async() => ({
           taskToken: token,
           input: JSON.stringify(input)
         })
       }),
       sendTaskFailure: (msg) => ({
         promise: () => {
-          t.is(msg.error, input.error.name);
-          t.is(msg.cause, input.error.message);
+          t.is(msg.error, 'Error');
```

Comment:
Why did this expectation change?

Reply:
Because the test lambda is now throwing an actual error, and not putting an object in the callback.